dashboard/src/actions/overviewActions.js

```diff
@@ -68,26 +62,23 @@ const initializeRuns = () => (dispatch, getState) => {
   data.forEach((item) => {
     item[CONSTANTS.IS_EDIT] = false;
     item[CONSTANTS.NAME_COPY] = item.name;
-    item[CONSTANTS.SERVER_DELETION_COPY] =
-      item.metadata[CONSTANTS.SERVER_DELETION];
+    item[CONSTANTS.SERVER_DELETION_COPY] = item.metadata.server.deletion;
 
     clearEditableFields(item);
     item[CONSTANTS.NAME_VALIDATED] = CONSTANTS.SUCCESS;
-    item[CONSTANTS.IS_ITEM_SEEN] = !!item?.metadata?.[CONSTANTS.DASHBOARD_SEEN];
+    item[CONSTANTS.IS_ITEM_SEEN] = !!item?.metadata?.global?.dashboard?.seen;
     item[CONSTANTS.IS_ITEM_FAVORITED] =
-      !!item?.metadata?.[CONSTANTS.USER_FAVORITE];
+      !!item?.metadata?.user?.dashboard?.favorite;
   });
 
   const savedRuns = data.filter(
-    (item) => item.metadata[CONSTANTS.DASHBOARD_SAVED]
-  );
-  const newRuns = data.filter(
-    (item) => !item.metadata[CONSTANTS.DASHBOARD_SAVED]
+    (item) => item.metadata.global?.dashboard.saved
   );
+  const newRuns = data.filter((item) => !item.metadata.global?.dashboard.saved);
 
   const expiringRuns = data.filter(
     (item) =>
-      findNoOfDays(item.metadata[CONSTANTS.SERVER_DELETION]) <
+      findNoOfDays(item.metadata.server.deletion) <
```

Comment:
You created `item[CONSTANTS.SERVER_DELETION_COPY]` above: if it's not going to be used here, maybe just use this direct access anywhere you might have used it, and drop the copy?